package-lock.json

```diff
@@ -663,6 +663,7 @@
       "resolved": "https://registry.npmjs.org/boom/-/boom-0.4.2.tgz",
       "integrity": "sha1-emNune1O/O+xnO9JR6PGffrukRs=",
       "dev": true,
+      "optional": true,
```

Comment:
I don't know what this (and the same below) means/does, but I'm sure it's fine...